### PR TITLE
Disable knob animation on touch devices 

### PR DESCRIPTION
**Pull request by karimJWP**
disables knob animation on touch devices 

Fixes #
JW7-4242

### CODE REVIEW

File: src/css/skins/seven.less [hunk 1]
```diff
@@ -50,14 +50,16 @@
         -moz-osx-font-smoothing: grayscale;
     }
 
-    .jw-slider-container {
-        .jw-knob {
-            display: none;
-        }
-
-        &:hover {
+    &:not(.jw-flag-touch) {
```
Comment (robwalch) [inline src/css/skins/seven.less]: This increases specificity, which should be avoided.

If the goal is to override existing ruled on touch devices, add a new `.jw-flag-touch` that overrides the existing rule.

Comment (karimJWP) [inline src/css/skins/seven.less]: addressed

**Comment by robwalch** [comment]
@karimJWP please rebase to address the build error. It looks like the commercial build is still pulling down the old template files and that's why the build fails.

**Comment by karimJWP** [comment]
@robwalch PR rebased

**Comment by egreaves** [comment]
@karimJWP the knob always shows on desktop in `timeSliderAbove` mode: 
screen shot 2017-04-10 at 10 41 34 pm

**Pull request merged**